homeassistant/components/renson/sensor.py

```diff
@@ -294,9 +284,9 @@ def _handle_coordinator_update(self) -> None:
         if self.raw_format:
             self._attr_native_value = value
         elif self.entity_description.device_class == SensorDeviceClass.ENUM:
-            self._attr_native_value = OPTIONS_MAPPING.get(
-                self.api.parse_value(value, self.data_type), None
-            )
+            self._attr_native_value = self.api.parse_value(
+                value, self.data_type
+            ).lower()
```

Comment:
Is the function always returning a string?

```suggestion
            value = self.api.parse_value(value, self.data_type).lower()
            
            if value in self.entity_description.options:
                return value
               
            return None
```

Reply:
Yes it will always return a string value

Reply:
Thanks for clearing that up @jimmyd-be 👍